netlify-cms-academic/static/admin/cms-preview-templates/post.js

```diff
@@ -0,0 +1,20 @@
+var PostPreview = createClass({
+  render: function() {
```

Comment:
Please refactor this function to improve the code structure and make it easier for contributors to follow.

Also, please use `let` in this context, curly brackets around IF statements, and spaces after `if` etc.

Reply:
Done.

Reply:
Done.